recipes/ssake/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "SSAKE" %}
+{% set version = "4.0" %}
+{% set sha256 = "873361ee4121e5667a2db5329ed739410e63eb1db39a76f13f6951b85b5a9a23" %}
+
+about:
+  home: https://github.com/warrenlr/SSAKE
+  license: GPL-2.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: SSAKE is a genomics application for de novo assembly of millions of very short DNA sequences.
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  sha256: {{ sha256 }}
+  url: https://github.com/warrenlr/{{ name }}/archive/v{{ version }}.tar.gz
+
+requirements:
+  run:
+    - perl-statistics-descriptive
```

Comment:
Do you also need perl here?

Reply:
"perl" is installed as part of "perl-statistics-descriptive". Isn't this enough? If I specify "perl" is an independent requirement, there is a version incompatibility with the other packages, and the build fails.